src/Window.cpp

```diff
@@ -85,34 +85,37 @@ struct Gosu::Window::Impl
 {
     bool fullscreen;
     double update_interval;
-    
+    bool resizable;
+
     // A single `bool open` is not good enough to support the tick() method: When close() is called
     // from outside the window's call graph, the next call to tick() must return false (transition
     // from CLOSING to CLOSED), but the call after that must return show the window again
     // (transition from CLOSED to OPEN).
     enum { CLOSED, OPEN, CLOSING } state = CLOSED;
-    
+
     unique_ptr<Graphics> graphics;
     unique_ptr<Input> input;
 };
 
-Gosu::Window::Window(unsigned width, unsigned height, bool fullscreen, double update_interval)
+Gosu::Window::Window(unsigned width, unsigned height, bool fullscreen, double update_interval, bool resizable)
 : pimpl(new Impl)
 {
     // Even in fullscreen mode, temporarily show the window in windowed mode to centre it.
     // This ensures that the window will be centred correctly when exiting fullscreen mode.
     // Fixes https://github.com/gosu/gosu/issues/369
     // (This will implicitly create graphics() and input(), and make the OpenGL context current.)
+    SDL_SetWindowResizable(shared_window(), (SDL_bool)resizable);
```

Comment:
Travis's SDL must be lower than SDL 2.0.5... Build fails here.

Reply:
There's an issue for upgrading Gosu's CI environment #478 — but Ubuntu 16.04 still ships with 2.0.4, so that won't help. To make the CI happy, this should be wrapped in an `#if` block like other "recent" SDL features.